types/classnames/index.d.ts

```diff
@@ -1,22 +1,23 @@
-// Type definitions for classnames
+// Type definitions for classnames 2.2
 // Project: https://github.com/JedWatson/classnames
-// Definitions by: Dave Keen <http://www.keendevelopment.ch>, Adi Dahiya <https://github.com/adidahiya>, Jason Killian <https://github.com/JKillian>
+// Definitions by: Dave Keen <http://www.keendevelopment.ch>
```

Comment:
Multilines are allowed. Moreover max line length is 200. https://github.com/mradamczyk/DefinitelyTyped#edit-an-existing-package